### DISCUSSION TITLE
How to use copilot to create a GitHub app

### DISCUSSION THREAD

**Discussion by Charliebrickman75** [community/community#48375] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I am self educating myself with GitHub and copilot. Does anyone have any suggestions as to how it can be used to create a GitHub app for NFT creations

**Comment by Charliebrickman75** [comment]
Use chat got to write code

**Comment by Charliebrickman75** [comment]
Meant chat got

**Comment by williamhnyohei** [comment]
You have two solid options for generating NFTs:

Quick & Easy:
If you're ready to invest a little money, try [NFT-Inator](https://nft-inator.com/?gad_source=1&gclid=Cj0KCQjwhMq-BhCFARIsAGvo0KdH7RI80IXjoe0TEiGPM5yymVA5E-SzJ4dGCC-LTlvFvTQyV_Tio_IaAsniEALw_wcB) for a turnkey solution.

DIY Approach:
Prefer a hands-on method? Fork the [NFT-Art-Generator](https://github.com/AppsusUK/NFT-Art-Generator) repo and use GitHub Copilot to understand the base code and help you generate your own NFT project.
